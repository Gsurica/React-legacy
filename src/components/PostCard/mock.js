export const postCardMock = {  
  title: 'something',
  body: 'body 1',
  cover: 'https//site.com.br',
  id: 1
}