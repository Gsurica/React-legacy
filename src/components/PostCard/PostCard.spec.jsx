import { render, screen } from "@testing-library/react"
import { PostCard } from "."
import { postCardMock } from './mock'

const props = postCardMock

describe('<PostCard />', () => {
  it('should render PostCard correctly', () => {
    render(<PostCard {...props} />)
    
    expect(screen.getByRole('img', { name: /something/i })).toHaveAttribute('src', 'https//site.com.br')
    expect(screen.getByRole('heading', { name: /something/i })).toBeInTheDocument()
  })
})