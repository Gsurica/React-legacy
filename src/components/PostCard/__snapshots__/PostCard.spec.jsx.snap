// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostCard /> should match the component snapshot 1`] = `
<div
  class="post"
>
  <img
    alt="something"
    src="https//site.com.br"
  />
  <div
    class="post-card"
  >
    <h2>
      something
    </h2>
    <p>
      body 1
    </p>
  </div>
</div>
`;
